src/Sylius/Bundle/UiBundle/Resources/views/Form/theme.html.twig

```diff
@@ -40,7 +40,7 @@
 
 {% block money_widget -%}
     <div class="ui labeled input">
-        <div class="ui label">{{ sylius_currency_symbol() }}</div>
+        <div class="ui label">{{ shopperContext.currencyCode|sylius_currency_symbol }}</div>
```

Comment:
shopper? chopper? harley? Why don't you use a `shop` prefix instead of `shopper`?